virttest/utils_test/__init__.py

```diff
@@ -1808,40 +1782,146 @@ def load_stress_tool(self):
         """
         load stress tool in guest
         """
+        if not self.install():
+            error_context.context("stress installation failed.")
+            return False
         session = self.get_session()
-        command = run_autotest(self.vm, session, self.control_path,
-                               None, None, self.params, copy_only=True,
-                               control_args=self.stress_args)
-        session.cmd("%s &" % command)
-        logging.info("Command: %s", command)
-        running = utils_misc.wait_for(self.app_running, first=0.5, timeout=60)
-        if not running:
-            raise StressError("Stress tool %s isn't running"
-                              % self.stress_type)
+        options = self.params.get('stress_args',
+                                  '--cpu 2 --io 1 --vm 1 --vm-bytes 128M')
+        try:
+            session.cmd_output_safe('nohup stress %s &' % options)
+        # The background process sometimes does not return to
+        # terminate, if timeout, send a blank line afterward
+        except aexpect.ShellTimeoutError:
+            session.cmd_output_safe('')
```

Comment:
it can be rather '\n'

Reply:
Thanks @balamuruhans Inside session.cmd_output_safe('') it add up a '\n' as 'linesep', so send '\n' will be transferred to '\n\n'. So I suppose that a blank command will be okay.